test/functional/scan/test-scan-exclusive-tbb.cpp

```diff
@@ -0,0 +1,23 @@
+
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+// Copyright (c) 2016-20, Lawrence Livermore National Security, LLC
+// and RAJA project contributors. See the RAJA/COPYRIGHT file for details.
+//
+// SPDX-License-Identifier: (BSD-3-Clause)
+//~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~//
+
+#include "test-scan-exclusive.hpp"
+
+#if defined(RAJA_ENABLE_TBB)
+
+// TBB policy types to test
+using TBBExecTypes = list< RAJA::tbb_for_exec >;
```

Comment:
RAJA::tbb_for_static and RAJA::tbb_for_dynamic?

Reply:
Thanks. I'll add that.

Reply:
@mdavis36 actually, tbb_for_static is not exposed as a user policy. tbb_for_exec is an alias to the TBB static policy.

Reply:
I thought that tbb_for_exec just called the default of tbb_for_static where Chunksize = 1 and that tbb_for_static was for using a user defined Chunksize.

Reply:
I thought so too. But, when I added it, the code didn't compile. There is no use of tbb_for_static in or for all unit tests.  Did you get it to work for the forall tests? I will skip it for now and add an issue that we can look into.

Reply:
Wait. I think I was using it wrong.  Let me try again.

Reply:
I did, it wont compile without the template argument, I am launching it as tbb_for_static<8>

Reply:
Yup. I screwed up. I added a few tests for tbb_for_static.